### PR TITLE
Feature/8 mvc

### PR DESCRIPTION
**Pull request by EdrinHasaj**

### CODE REVIEW

File: BoggleModel.java [hunk 1]
```diff
@@ -109,12 +113,116 @@ private String randomizeLetters(int size){
...
+        }
+
+        private String gameMode;
+
+        GameMode(final String gameMode) {
+            this.gameMode = gameMode;
+        }
+    }
+
+    public void addHumanWord(String word){
```
Comment (Damon-D-Ma) [inline BoggleModel.java:144]: how do you check for invalid or duplicate words? Or are we changing the game such that the turn ends only when the timer runs out?

Comment (Damon-D-Ma) [reply BoggleModel.java:144]: misunderstanding resolved

**Review by Damon-D-Ma — APPROVED** [review]
Code looks like it is in a workable state and is ready to be merged into develop branch.

File: BoggleStats.java [hunk 2]
```diff
@@ -158,17 +158,17 @@ public void summarizeRound() {
...
         return ret;
     }
     public String getHCGameSummary(){
         String ret = "";
         ret += "Player 1 Total Score: " + this.p1ScoreTotal + "\n";
-        ret += "Player 1 Average Words: " + this.p1AverageWords + "\n";
+        ret += "Player 1 Total Words: " + this.p1AverageWords + "\n";
         ret += "Computer Total Score: " + this.cScore + "\n";
-        ret += "Computer Average Words: " + this.cAverageWords + "\n";
+        ret += "Computer Total Words: " + this.cAverageWords + "\n";
```
Comment (Teminix) [inline BoggleStats.java:171]: Good fix

File: BoggleModel.java [hunk 3]
```diff
@@ -109,12 +113,116 @@ private String randomizeLetters(int size){
...
+        for (String word : all_words.keySet()) {
+            if(!this.gameStats.getPlayer1Words().contains(word) && !this.gameStats.getPlayer2Words().contains(word)) {
+                this.gameStats.addWord(word, BoggleStats.Player.Computer);
+            }
+        }
+    }
+
+    public String getGameSummaryString(){
+        this.gameStats.endRound();
+        if(gameMode == GameMode.HumanHuman) {
```
Comment (Teminix) [inline BoggleModel.java:210]: Could make your if conditions in one line in the _next_ sprint, no need to worry about it now. Plus this is just a suggestion.
```java
if(gameMode == GameMode.HumanHuman) return this.gameStats.getHHGameSummary();
```

**Review by Teminix — COMMENTED** [review]
Code looks good for merging to dev.

**Pull request merged**